applications/nrf5340_audio/tools/buildprog/buildprog.py

```diff
@@ -109,6 +110,16 @@ def __build_cmd_get(core: Core, device: AudioDevice, build: BuildType, pristine,
         if options.nrf21540:
             device_flag += " -DSHIELD=nrf21540ek_fwd"
 
+        if options.user_bt_name:
```

Comment:
There needs to be a guard here. What happens if a user supplies both --user_bt_name" and "--custom_bt_name"?
The simplest way to do this is to generate precedence by checking `if options.custom_bt_name is not None:` first and then do an `else if` for `options.user_bt_name:`

Reply:
I think we should just raise an error and exit if both arguments are given, as giving both is for sure a usage error.